app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java

```diff
@@ -0,0 +1,20 @@
+package se.inera.intyg.minaintyg.util.html;
+
+import se.inera.intyg.minaintyg.integration.api.certificate.model.value.CertificateQuestionValueList;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.value.CertificateQuestionValueTable;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.value.CertificateQuestionValueText;
+
+public class CertificateQuestionValueHTMLFactory {
```

Comment:
Add private constructors and throw new IllegalStateException("Utility class"); if used.